backend-rust/README.md

```diff
@@ -114,38 +114,20 @@ This project has some dependencies tracked as Git submodules, so make sure to in
 git submodule update --init --recursive
 ```
 
-### Running the database server
-
-Both services depend on having a PostgreSQL server running, this can be done in several ways, but it can be done using [docker](https://www.docker.com/) with the command below:
-
-```
-docker run -p 5432:5432 -e 'POSTGRES_PASSWORD=example' -e 'POSTGRES_DB=ccd-scan' postgres:16
-```
-
-Alternatively set up the database from the `docker-compose` file with the command below:
+### Initializing a database from scratch
 
 ```
-docker compose up
+make setup && make
 ```
 
-### Initializing a database
+where `make setup` is a one time command initialising the password to be used and `make` starts the database and initialises it
 
-Then set the environment variable `DATABASE_URL` pointing to the location of the SQL database, this can be done by creating a `.env` file within this directory.
-Example:
+### Reuse an already initialised database
 
 ```
-# Postgres database connection used by sqlx-cli and this service.
-DATABASE_URL=postgres://postgres:example@localhost/ccd-scan
+make setup-env-with-password && make
 ```
 
-With the environment variable `DATABASE_URL` set, use the `sqlx` CLI to setup the database and tables and run all the migrations:
-
-```
-sqlx migrate run
-```
-
-The project can now be built using `cargo build`
-
```

Comment:
If someone is running the make command you will be initialising the project

Would you rather have the information explicitly pointed out on how to do it without using the makefile?

Personally I would assume that if someone diverges from the provided flow then they better know what they are doing. 

If you disagree I can just add a section called Make a manually configured database.

Perhaps make all ways of init a subsection.

To me it will just be a lot of ways of achieving the same thing, while I would prefer the readme to be more concise.

Reply:
double checked and I think the readme mentioned `cargo run` commands which compile the project as well, so seems ok for me if we don't explicitly mentioned how to build the project.